### PR TITLE
Adds new heatmap example

### PR DESCRIPTION
**Pull request by srtena**
### Description

With the new experimental `HeatmapTileLayer` available in @deck.gl/carto, we can now easily build beautiful heatmaps from large point-based datasets, benefitting from CARTO dynamic tiling for scalability and performance.

This example tries to showcase how any developer can use the HeatmapTileLayer easily :) 

### TO DO

- [x] Use the same API Access Token or unify with the rest of examples
- [x] Confirm final documentation links

### CODE REVIEW

File: labels/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
       rel="stylesheet"
     />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>CARTO for deck.gl - Labels example wiht Collision Filter Extension</title>
+    <title>CARTO + deck.gl - Labels example wiht Collision Filter Extension</title>
```
Comment (felixpalmer) [inline labels/index.html:24]:
```suggestion
    <title>CARTO + deck.gl - Labels example with Collision Filter Extension</title>
```

File: dynamic-heatmap/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Example: Dynamic Heatmap using a 587k points worldwide fires dataset
```
Comment (felixpalmer) [inline dynamic-heatmap/README.md:1]: Why do we need the word "Dynamic", can't we just say "Heatmap"?

File: dynamic-heatmap/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CARTO + deck.gl - Dynamic Heatmap from points</title>
+  <body>
+    <div id="app">
+      <div id="map"></div>
+      <canvas id="deck-canvas"></canvas>
+      <div id="story-card">
+        <p class="overline">✨👀 You're viewing</p>
```
Comment (felixpalmer) [inline dynamic-heatmap/index.html:30]: Generally this looks good, you don't feel like adding sliders for the intensity and/or radius?

**Review by felixpalmer — APPROVED** [review]

File: dynamic-heatmap/index.html (hunk 3 above)
Comment (srtena) [reply dynamic-heatmap/index.html]: I wasn't sure about the value of interactivity but hell yeah why not who doesn't love sliders :)

File: dynamic-heatmap/README.md (hunk 2 above)
Comment (srtena) [reply dynamic-heatmap/README.md]: @felixpalmer let me know what you think now

**Review by felixpalmer — APPROVED** [review]

**Pull request merged**